packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -1540,6 +1540,24 @@ new rds.DatabaseCluster(this, 'Cluster', {
 });
 ```
 
+## Importing existing DatabaseInstance
+
+### Lookup DatabaseInstance by instanceIdentifier
+
+You can lookup an existing DatabaseInstance by its instanceIdentifier using `DatabaseInstance.fromLookup()`.  This method returns an `IDatabaseInstance`.
+
+Here's how `DatabaseInstance.fromLookup()` can be used:
+
+```ts
+declare const myUserRole: iam.Role;
+
+const dbFromLookup = rds.DatabaseInstance.fromLookup(this, 'dbFromLookup', {
+  instanceIdentifier: 'instanceId',
+});
+
+// Grant a connection
+dbFromLookup.grantConnect(myUserRole, 'my-user-id');
+
```

Comment:
```suggestion
```ts
declare const myUserRole: iam.Role;

const dbFromLookup = rds.DatabaseInstance.fromLookup(this, 'dbFromLookup', {
  instanceIdentifier: 'instanceId',
});

// Grant a connection
dbFromLookup.grantConnect(myUserRole, 'my-user-id');
```
s
```